recipe/build.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 
-# Make sure cf_units can find the udunits library.
 SITECFG=cf_units/etc/site.cfg
```

Comment:
I recon we can remove the site.cfg file for Linux & OSX.

Reply:
I left it there for "symmetry" no strong opinions and glad to remove it.